src/config/hdc.ts

```diff
@@ -11,7 +11,7 @@ export const FRAMEKM_ROOT_FOLDER = '/mnt/data/framekm';
 export const UNPROCESSED_FRAMEKM_ROOT_FOLDER = '/mnt/data/unprocessed_framekm';
 export const RAW_DATA_ROOT_FOLDER = '/mnt/data/raw';
 export const STREAM_REQUEST_FOLDER = '/mnt/data/request';
-export const DB_PATH = '/mnt/data/data-logger.v1.4.2.db';
+export const DB_PATH = '/mnt/data/data-logger.v1.4.3.db';
```

Comment:
No need to uptick DB version,
new table creation is a safe change for other data!

Reply:
reverted the change